bot/exts/utilities/color.py

```diff
@@ -0,0 +1,337 @@
+import colorsys
+import json
+import logging
+import re
+from io import BytesIO
+
+from PIL import Image, ImageColor
+from discord import Embed, File
+from discord.ext import commands
+from rapidfuzz import process
+
+from bot.bot import Bot
+from bot.constants import Colours
+
+
+logger = logging.getLogger(__name__)
+
+
+ERROR_MSG = """The color code {user_color} is not a possible color combination.
+\nThe range of possible values are:
+\nRGB & HSV: 0-255
+\nCMYK: 0-100%
+\nHSL: 0-360 degrees
+\nHex: #000000-#FFFFFF
+"""
+
+COLOR_JSON_PATH = "bot/resources/utilities/ryanzec_colours.json"
+with open(COLOR_JSON_PATH) as f:
```

Comment:
```suggestion
with open("bot/resources/utilities/ryanzec_colours.json") as f:
```
I understand why it is used, but we can just use this.